src/Illuminate/Container/Container.php

```diff
@@ -548,21 +548,21 @@ public function tagged($tag)
     /**
      * Alias a type to a different name.
      *
-     * @param  string  $abstract
      * @param  string  $alias
+     * @param  string  $abstract
      * @return void
      *
      * @throws \LogicException
      */
-    public function alias($abstract, $alias)
+    public function alias($alias, $abstract)
```

Comment:
Didn't we recently swap these around?

Reply:
There are?  
Where can I see the changes you are talking about ？

Reply:
Good memory, but the `Container.php` has always been in this order since the first commit. That was the AliasLoader: https://github.com/laravel/framework/pull/33294

If you change the parameter order here, you'll have to make the same change in the Container Contract. https://github.com/laravel/framework/blob/9.x/src/Illuminate/Contracts/Container/Container.php#L27